cypress/integration/accessibility_OnLinkNavigationProvider_spec.js

```diff
@@ -7,4 +7,15 @@ describe('OnLinkNavigationProvider Accessibility check', () => {
   it('Tests accessibility on the OnLinkNavigationProvider page', () => {
     cy.checkA11y();
   });
+
+  // Keep disabled until converting to new Code editor
+  cy.configureAxe({
+    rules: [
+      {
+        id: 'color-contrast',
+        enabled: false,
+      },
+    ],
+  });
```

Comment:
These lines should go above line 8, and then you don't need to repeat cy.checkA11y();

Reply:
Done!

Reply:
sooo this should look like:
```

it('Tests accessibility on the OnLinkNavigationProvider page', () => {
  cy.configureAxe({
    rules: [
      {
        id: 'color-contrast',
        enabled: false,
      },
    ],
  });

    cy.checkA11y();
  });
  
```
  Just like your Upsell one now